index.html

```diff
@@ -136,7 +136,7 @@ <h3>Top Tracks</h3>
       <main class="search-profile">
         <h1>Album Profile</h1>
         <figure class="search-pic">
-          <img src="" alt="" id="album-pic" />
+          <a id="artist" class="image waves-effect waves-light modal-trigger" href="#artist-modal"><img title="" src="" alt="" id="album-pic" /></a>
```

Comment:
Accessibility, 'title' needs to be more descriptive

Reply:
title is now more descriptive